cfme/fixtures/v2v_fixtures.py

```diff
@@ -337,16 +335,17 @@ def __set_conversion_instance_for_rhev_ui(appliance, source_provider,  # noqa
     with open(temp_file.name, 'w') as f:
         f.write(__conversion_data(rhv_provider)["private_key"])
     conv_host_key = temp_file.name
-
+    rhev_hosts = [h.name for h in rhv_provider.hosts.all()]
     for host in rhev_hosts:
-        __configure_conversion_host_ui(appliance,
-                                       rhv_provider, host.name, "Default",
-                                       conv_host_key, transformation_method,
-                                       vmware_ssh_key)
+        conv_host = configure_conversion_host_ui(appliance, rhv_provider,
+                                                 host, "Default",
+                                                 conv_host_key, transformation_method,
+                                                 vmware_ssh_key)
```

Comment:
Q: why we are creating list of names; I think old logic also fine.

Reply:
@digitronik : to assert if each host is configured or not in the next line .